etc/module.xml

```diff
@@ -24,7 +24,7 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
 
-    <module name="Adyen_Payment" setup_version="7.3.2">
+    <module name="Adyen_Payment" setup_version="7.3.3">
```

Comment:
Are there any changes specific to the database for this to be modified?

Reply:
No but we still always keep it aligned with the version, according to our docs. This time magento will not run any upgrade script